examples/blog-with-auth/apps/backend/migrate.sh

```
@@ -1,8 +1,25 @@
+#    We use "$PROJECT_ROOT" so these become /app/apps/backend/...
+#    Adjust the relative part 'apps/backend/...' to match your actual structure
+SCHEMA_PATH="$PROJECT_ROOT/prisma/schema.prisma"
+CONFIG_PATH="$PROJECT_ROOT/prisma.config.ts"
+
 echo "Deploying with Prisma version: $PRISMA_VERSION"
-npx -y prisma@$PRISMA_VERSION migrate deploy
+echo "Using Schema: $SCHEMA_PATH"
+echo "Using Config: $CONFIG_PATH"
+
+# 4. Move to /tmp to trick npx into ignoring the local package.json
+cd /tmp
```

Shellcheck (0.11.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)